Easydict/NewApp/View/SettingView/Tabs/AboutTab.swift

```diff
@@ -45,8 +46,7 @@ struct AboutTab: View {
         }
     }
 
-    @AppStorage("EZConfiguration_kAutomaticallyChecksForUpdatesKey")
-    private var autoChecksForUpdates = false
+    @Default(.autoCheckForUpdates) private var autoChecksForUpdates
```

Comment:
Maybe we do not need to save `autoCheckForUpdates` to defaults.

Refer Sparkle docs:

```objc
/**
 A property indicating whether or not to check for updates automatically.
 
 By default, Sparkle asks users on second launch for permission if they want automatic update checks enabled
 and sets this property based on their response. If `SUEnableAutomaticChecks` is set in the Info.plist,
 this permission request is not performed however.
 
 Setting this property will persist in the host bundle's user defaults.
 Hence developers shouldn't maintain an additional user default for this property.
 Only set this property if the user wants to change the default via a user settings option.
 Do not always set it on launch unless you want to ignore the user's preference.
 For testing environments, you can disable update checks by passing `-SUEnableAutomaticChecks NO`
 to your app's command line arguments instead of setting this property.
 
 The update schedule cycle will be reset in a short delay after the property's new value is set.
 This is to allow reverting this property without kicking off a schedule change immediately
 */
@property (nonatomic) BOOL automaticallyChecksForUpdates;
```

Reply:
How about set `SUEnableAutomaticChecks` to YES in the Info.plist?

This is old code:

https://github.com/tisfeng/Easydict/blob/2df80b922865103b24d19a22b541b89c3218bf3e/Easydict/Feature/Configuration/EZConfiguration.m#L146

Reply:
@Jerry23011 Try binding 
```swift
var autoChecksForUpdates: Binding<Bool> {
    .init(get:set)
}
```

Reply:
This code doesn't work, please check it.

Reply:
Fixed with some ugly codes in 28c9c02. Any suggestion?

Reply:
It works, we can use this code if there is no better solution.

Reply:
Thanks @CanglongCl